threedi_modelchecker/checks/raster.py

```diff
@@ -300,6 +300,20 @@ def description(self):
         return f"{self.column_name} has values {' and/or '.join(parts)} or is empty"
 
 
+class RasterCompressionUsedCheck(BaseRasterCheck):
+    """Checks whether compression was used for the raster"""
+
+    def is_valid(self, path: str, interface_cls: Type[RasterInterface]):
+        with interface_cls(path) as raster:
+            if not raster.is_valid_geotiff:
+                return True
+            if raster.compression != "NONE":
+                return True
+
```

Comment:
Personally I'd add a `return False` to the end. This make it more clear, to me, that this functions tests a list of conditions and if none match than `is_valid` will return `False`.

Reply:
Great, given the explanation above that is actually much clear; the structure of the code then becomes:

* Ensure check should be performed on this path: if not return `False`
* Perform check

Reply:
I saw it slightly differently
Ensure check can be performed on this path; if not, return `True` so the check doesn't raise an exception
Perform check